Source/CarthageKit/Cartfile.swift

```diff
@@ -131,7 +131,7 @@ public func duplicateProjectsIn(_ cartfile1: Cartfile, _ cartfile2: Cartfile) ->
 /// checked out for each dependency.
 public struct ResolvedCartfile {
 	/// The dependencies listed in the Cartfile.resolved.
-	public var dependencies: [Dependency<PinnedVersion>]
+	public var dependencies: Set<Dependency<PinnedVersion>>
```

Comment:
I want to get rid of this entirely and use the dictionary below (with a similar change in `Cartfile`), but this is a good first step.